woocommerce/woocommerce-ios
# Coupons: Update coupon details screen following the updated design

Closes: #6487 


### Description

This PR updates the layout and contents of the coupon details screen to match with the latest design.

The screen now displays details of coupons in plain texts and only includes details that are available. I've added a few more fields for coupons in the summary section that were not included in the design to provide more information.

The amount section is pushed to the bottom to make place for the coupon code, expiry status and summary section on the top.

### Testing instructions

- Make sure that you have coupons enabled in your test store and in the experimental features section in the app
- Set up a few coupons for your test store with different attributes.
- Launch the app and navigate to Menu > Coupons. Select the items in the list - notice that the layout and contents should be updated to match the new design.

### Screenshots

| Before | After |
| ----- | ----- |
| <img src="https://user-images.githubusercontent.com/5533851/160333277-7faf4984-38b3-4a6c-9d7f-e247d21404f9.PNG" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/160333334-e44b0863-6b38-4f5f-b193-1bd14fbb6c99.png" width=320 /> |
| <img src="https://user-images.githubusercontent.com/5533851/160333403-0c13864b-058c-40d2-819f-7dc64ee7bb7c.PNG" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/160333452-191d29f0-c278-44eb-80f9-b1200dbcaa5c.png" width=320 /> |

When a coupon has more details:
<img src="https://user-images.githubusercontent.com/5533851/160771852-02ea3107-9959-4643-a4e3-6aae5e92b29b.png" width=320 />

---
